rules/symfony4/tests/Rector/ClassMethod/ConsoleExecuteReturnIntRector/Fixture/skip_ternary_return_int.php.inc

```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace Rector\Symfony4\Tests\Rector\ClassMethod\ConsoleExecuteReturnIntRector\Fixture;
+
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+final class SkipTernaryReturnInt extends Command
+{
+    public function execute(InputInterface $input, OutputInterface $output): int
+    {
+        return rand(0, 1) ? 0 : 1;
+    }
```

Comment:
What about this case?

```php
$yes = 0;
$no = 1;
return rand(0, 1) ? $yes : $no;
```

Reply:
implemented https://github.com/rectorphp/rector/pull/5735/commits/6be8c8fd525ebb5c4a058dca9b558a1f63be4dd8